tests/test_flows.py

```diff
@@ -454,6 +454,19 @@ def test_ap_config_bss_tear_down(self):
             if vap.ssid != b'N/A':
                 self.fail('Wrong SSID: {vap.ssid} instead torn down'.format(vap=vap))
 
+    def test_message_to_radio_cli(self): 
+        actions = ["ACTION_BML_MESSAGE_TO_RADIO_REQUEST", "ACTION_CONTROLLER_MESSAGE_TO_RADIO_REQUEST"]
+
+        for i in range(0, 2):
+            debug("Testing message received at point {i}".format(i))
+            env.beerocks_cli_command("bml_message_to_radio {agents[i].mac}".format(env.agents[0].mac))
+            # Wait a bit for the message
+            time.sleep(1)
+            # Check controller
+            self.check_log(env.controller.mac, actions[0])
```

Comment:
no need for `actions` code will be more readable  like this 
```suggestion
            self.check_log(env.controller.mac, ACTION_BML_MESSAGE_TO_RADIO_REQUEST)
```